cmd/dlv/dlv_test.go

```diff
@@ -180,17 +180,17 @@ func testOutput(t *testing.T, dlvbin, output string, delveCmds []string) (stdout
 
 	// ignore "dlv debug" command error, it returns
 	// errors even after successful debug session.
-	cmd.Wait()
+	_ = cmd.Wait()
```

Comment:
Let's not do this, it's everywhere and it looks ugly.

Reply:
Hah, agreed, automated change. Removed.